functions/src/services/GoogleSheets.ts

```diff
@@ -103,8 +103,8 @@ export default class GoogleSheets {
   }
 
   public async getRow(rowNumber: number): Promise<any> {
-    this.connect();
-    this.getHeaders();
+    await this.connect();
+    await this.getHeaders();
```

Comment:
This may be a good suggestion, however this is not related to your current task. See point 5 of https://trello.com/c/rkJKKl5d/107-git-conventions.

Reply:
I was getting error during linting that's why i fixed that.